packages/storybook/stories/CallComposite/QuickstartPSTNDocs.stories.mdx

```diff
@@ -0,0 +1,130 @@
+import { Meta, Source } from '@storybook/addon-docs';
+import QuickstartPSTNSnippet from '!!raw-loader!./snippets/QuickstartPSTN.snippet.tsx';
+import { SingleLineBetaBanner } from '../BetaBanners/SingleLineBetaBanner';
+import { DetailedBetaBanner } from '../BetaBanners/DetailedBetaBanner';
+import { StartPSTNCall } from './StartPSTNCall.stories';
+import { CallComposite } from '@azure/communication-react';
+import { defaultCallCompositeHiddenControls, controlsToAdd } from '../controlsUtils';
+
+<Meta
+  id="quickstarts-pstn"
+  title="Composites/CallComposite/PSTN Call/PSTN Get Started"
+  parameters={{ previewTabs: { canvas: { disable: true, hidden: true } } }}
+/>
+
+# Getting Started With PSTN
+
+<DetailedBetaBanner />
+
+The UI Library CallWithChat Composite and Calling Composite supports PSTN by enabling developers to dial and call phone numbers through
+Azure Communication Services call service. The UI Library renders a dialpad that supports DTMF (dual tone multi-frequency) and other
+features such as being put on hold and dialing in and removing users from an existing call.
+
+More specifically, the following features are supported:
+
+1. [DTMF](./?path=/docs/ui-components-dialpad--dialpad) (dual tone multi-frequency)
+2. [Dialpad UI](./?path=/docs/ui-components-dialpad--dialpad)
+3. [On/Off hold features](./?path=/docs/composite-adapters--page)
+4. 1 to 1 participant PSTN calls
+5. Multi participant PSTN calls
+6. Dialing/Adding PSTN participants into an existing call
+7. Removing PSTN participants from a call
+
+_Unfortunately, users are not able to join into an existing call through a copy invite link for 1:N and PSTN scenarios at this time._
+
+## Download the Code
+
+You can find the completed code for this quickstart here: [Get started with Composites](https://github.com/Azure-Samples/communication-services-javascript-quickstarts/tree/main/ui-library-quickstart-composites)
```

Comment:
hows the updates to these happening? are 1:N and PSTN getting their own samples at some point and this is going to be changed?

Reply:
We will be using the composite quickstarts and modifying the App.tsx.
I will change the text to reflect that! Good catch!